components/input/package.json

```diff
@@ -148,17 +147,44 @@
     "web components"
   ],
   "scripts": {
+    "build": "npm-run-all build:version build:sass sass:render bundler types",
+    "build:test": "npm-run-all test linters",
+    "build:release": "npm-run-all build build:test build:api build:docs bundler postinstall",
+    "build:ci": "npm-run-all sweep build:release",
+    "build:api": "wca analyze 'src/auro-input.js' --outFiles docs/api.md",
+    "build:demo": "npm-run-all build demo:rm:build demo:new:build demo:copy:index demo:copy:demo demo:update:index",
+    "build:dev:assets": "npm-run-all build:sass:component postCss:component sass:render",
+    "build:docs": "node ./node_modules/@aurodesignsystem/auro-library/scripts/build/generateDocs.mjs",
+    "build:sass": "npm-run-all build:sass:component postCss:component sass:render",
+    "build:sass:component": "sass --no-source-map src:src",
+    "build:version": "node scripts/version.mjs",
+    "build:watch": "nodemon -e scss,js --watch src --exec npm run build:dev:assets",
+    "bundler": "rollup -c",
+    "bundler:test": "rollup -c -w",
+    "scssLint": "stylelint \"./src/**/*.scss\"",
     "demo:copy:demo": "copyfiles -u 1 -V './demo/index.md' ./build/demo",
     "demo:copy:index": "copyfiles -u 1 -V './demo/index.html' ./build",
     "demo:new:build": "mkdir ./build && mkdir ./build/css && mkdir ./build/demo",
     "demo:rm:build": "rm -rf ./build",
     "demo:update:index": "node ./scripts/prepForBuild",
     "dev": "concurrently --kill-others 'npm run build:watch' 'npm run serve'",
     "dist:js": "copyfiles -u 1 -V './src/**/*.js' ./dist",
+    "esLint": "./node_modules/.bin/eslint src/**/*.js --ignore-pattern 'src/**/*Version.js'",
+    "linters": "npm-run-all scssLint esLint",
+    "preCommit": "node ./node_modules/@aurodesignsystem/auro-library/scripts/build/pre-commit.mjs",
     "postCss:component": "node ./node_modules/@aurodesignsystem/auro-library/scripts/build/postCss.mjs",
-    "postinstall": "node ./node_modules/@aurodesignsystem/auro-library/scripts/build/postinstall.mjs",
-    "sass:render": "sass-render --load-path=node_modules 'src/**/*.css' 'components/**/*.css' -t ./node_modules/@aurodesignsystem/auro-library/scripts/build/staticStyles-template.js",
+    "postinstall": "node packageScripts/postinstall.mjs",
```

Comment:
We should be using the auro-library scripts

Reply:
Currently no script that I use as a reference and also it's unclear what script will be remained after Turbo. I'd like to keep it as Auro-input until we know what we can do with Turbo.

Reply:
reverted the scripts section